### PR TITLE
fix(tags): add actions for tag consisting of spaces

### PR DESCRIPTION
**Pull request by HeyRoach**

### CODE REVIEW

**Comment by zolotyx** [comment]
Maybe just trim the value before saving it?

File: src/app/shared/directives/forbidden-values-validator.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export function forbiddenValuesValidator(
   forbiddenValues: Array<string>
 ): ValidatorFn {
   return (c: AbstractControl) => {
-    const isValid = !forbiddenValues.find(_ => _ === c.value);
+    const isValid = c.value && !forbiddenValues.find(_ => _ === c.value.trim());
```
Comment (tamazlykar) [inline src/app/shared/directives/forbidden-values-validator.ts]: screenshot from 2018-08-23 10-10-57

File: src/i18n/en.json [hunk 2]
```diff
@@ -1565,7 +1565,8 @@
     "TAG_KEY": "Key",
     "TAG_VALUE": "Value",
     "TAG_ALREADY_EXISTS": "Tag already exists",
-    "EDIT_TAG": "Edit tag"
+    "EDIT_TAG": "Edit tag",
+    "TAG_INVALID": "Value shouldn't started from space"
```
Comment (tamazlykar) [inline src/i18n/en.json]: Use name that describes a translation string

**Comment by tamazlykar** [comment]
@ershovaelena 
screenshot from 2018-08-23 10-15-02
Tags have different number of spaces but looks identical

**Comment by bwsw** [comment]
@HeyRoach Trimming is OK, by the way. No problem with that. @zolotyx thnx. 
@ershovaelena why you decided to go with error rather than just trimming?

**Comment by bwsw** [comment]
@HeyRoach don't change the behavior now. Just show an error if first char is a space char.

File: src/i18n/en.json [hunk 3]
```diff
@@ -1565,7 +1565,8 @@
     "TAG_KEY": "Key",
     "TAG_VALUE": "Value",
     "TAG_ALREADY_EXISTS": "Tag already exists",
-    "EDIT_TAG": "Edit tag"
+    "EDIT_TAG": "Edit tag",
+    "TAG_START_FROM_SPACE": "Value shouldn't started from space"
```
Comment (bwsw) [inline src/i18n/en.json]: @HeyRoach 
The tag keys and values shouldn't start with "space" symbol.

**Pull request merged**